packages/material-ui/src/Slider/ValueLabel.js

```diff
@@ -1,30 +1,43 @@
 import * as React from 'react';
-import clsx from 'clsx';
-import withStyles from '../styles/withStyles';
+import { ValueLabelUnstyled } from '@material-ui/unstyled';
```

Comment:
Should we prefix it?
```suggestion
import { SliderValueLabelUnstyled } from '@material-ui/unstyled';
```

Reply:
Good call, will rename it